### ISSUE TITLE
Button to add an input field / condition "Is any of"

### ISSUE THREAD

**Issue by maciejkaiser** [mistic100/jQuery-QueryBuilder#620]
Hello,

I need functionality "Is any of" as below (FilterBuilder from Devexpress)

![image](https://user-images.githubusercontent.com/12011503/34522626-cc78d37e-f093-11e7-925b-f4454147b871.png)

Is there any possibility to make custom filter with button to add an input fields and a quotation marks?

**Comment by mistic100** [comment]
Use the `in` operator with the `value_separator` filter property.
The user will be able to enter `value1, value2` and you will get an array `["value1", "value2"]`

**Comment by maciejkaiser** [comment]
Thank you for yours answer, but I need this add button functionality to creating many inputs in filter. It's possible to changing `nb_inputs` value or something ? I have no idea how to create similar mechanism... 

First step ( by clicking on this green Add Button ) 
![image](https://user-images.githubusercontent.com/12011503/34555870-4d06395a-f134-11e7-8769-4f7c4e99ecb5.png)

Second step 
![image](https://user-images.githubusercontent.com/12011503/34555903-7529f796-f134-11e7-9e12-133e7dc36427.png)

Third
![image](https://user-images.githubusercontent.com/12011503/34555918-8c0b043c-f134-11e7-96d2-242e040cea41.png)

And more...
![image](https://user-images.githubusercontent.com/12011503/34555937-a2084588-f134-11e7-80bc-02b0f04bf7d0.png)

**Comment by mistic100** [comment]
I gave an alternative solution (which IMHO totally works) because there is no simple solution.

If you want something more complex, declare the `filter.input` as a function (see to "Coordinates" example on the demo website), and also implement `filter.valueGetter` and `filter.valueSetter`

**Comment by mistic100** [comment]
You can also create a standalone jQuery plugin (which transform in single input into your complex interface) and enable it with `filter.plugin`

**Comment by maciejkaiser** [comment]
I made a simple solution, but I've used an option with a filter - not an operator.

https://jsfiddle.net/rp86qvfj/28/

But what I need is to create an **operator** named "is any of" which allows user to manipulate the number of inputs (like in this fiddle). The problem is thath I have no idea how to merge that filter with the operator. Operators can't use any other but _type, nb_inouts, multiple, apply_to_ etc.
